Scripts/DCS-BIOS/lib/modules/aircraft_modules/FA-18C_hornet.lua

```diff
@@ -157,10 +161,19 @@ FA_18C_hornet:addExportHook(function()
 	ufc = Module.parse_indication(6) or {}
 end)
 
+local comm_channel_cache = {}
+
 local function get_comm_channel(channel)
 	local display_value = channel == 1 and ufc.UFC_Comm1Display or ufc.UFC_Comm2Display
 
-	return comm_channel_map[display_value] or 0
+	if display_value == "d" then -- receiving(?), comm channel hasn't changed
+		return comm_channel_cache[channel] or 0
+	end
+
+	local value = comm_channel_map[display_value] or 0
+	comm_channel_cache[channel] = value
+
+	return value
```

Comment:
I'm new to Lua, so I may be a bit confused by the syntax here..

Why did you introduce the `comm_channel_cache`?

Reply:
This was added to fix a different control that this symbol breaks, which outputs a numeric value for the channel selected. As this symbol _I assume_ is shown for all channels, we want to make sure we're still reporting that the same integer value is selected as the channel selection hasn't changed.

At least, I assume. I don't know why this control was added or who (if anyone) currently uses it.